docs/sections/user_guide/cli/mode_config.rst

```diff
@@ -377,31 +372,73 @@ and additional supplemental YAML file ``values2.yaml`` with content
+     [2024-01-23T23:01:23]    DEBUG     empty: null
+     [2024-01-23T23:01:23]    DEBUG     greeting: Good Night
+     [2024-01-23T23:01:23]    DEBUG     message: '{{ (greeting + " " + recipient + " ") * repeat }}'
+     [2024-01-23T23:01:23]    DEBUG     recipient: Moon
+     [2024-01-23T23:01:23]    DEBUG     repeat: 2
+     [2024-01-23T23:01:23]    DEBUG [dereference] Accepting: 20240105
+     [2024-01-23T23:01:23]    DEBUG [dereference] Accepting: None
+     [2024-01-23T23:01:23]    DEBUG [dereference] Rendering: Good Night
+     [2024-01-23T23:01:23]    DEBUG [dereference] Rendered: Good Night
+     [2024-01-23T23:01:23]    DEBUG [dereference] Rendering: {{ (greeting + " " + recipient + " ") * repeat }}
+     [2024-01-23T23:01:23]    DEBUG [dereference] Rendered: Good Night Moon Good Night Moon
+     [2024-01-23T23:01:23]    DEBUG [dereference] Rendering: Moon
+     [2024-01-23T23:01:23]    DEBUG [dereference] Rendered: Moon
+     [2024-01-23T23:01:23]    DEBUG [dereference] Accepting: 2
      ...
-     [2024-01-10T21:43:58]    DEBUG Rendering: 2
-     [2024-01-10T21:43:58]    DEBUG Rendered: 2
-     [2024-01-10T21:43:58]    DEBUG Dereferencing, final value: {'values': {'date': 20240105, 'empty': None, 'greeting': 'Good Night', 'message': 'Good Night Moon Good Night Moon', 'recipient': 'Moon', 'repeat': 2}}
+     [2024-01-23T23:01:23]    DEBUG Dereferencing, final value:
+     [2024-01-23T23:01:23]    DEBUG   values:
+     [2024-01-23T23:01:23]    DEBUG     date: 20240105
+     [2024-01-23T23:01:23]    DEBUG     empty: null
+     [2024-01-23T23:01:23]    DEBUG     greeting: Good Night
+     [2024-01-23T23:01:23]    DEBUG     message: 'Good Night Moon Good Night Moon '
+     [2024-01-23T23:01:23]    DEBUG     recipient: Moon
+     [2024-01-23T23:01:23]    DEBUG     repeat: 2
 
-* Note that ``uw`` does not allow invalid conversions. For example, when attempting to generate an ``sh`` config from a depth-2 ``yaml``:
+.. note:: Combining configs with incompatible depths is not supported. ``ini`` and ``nml`` configs are depth-2, as they organize their key-value pairs (one level) under top-level sections or namelists (a second level). ``sh`` configs are depth-1, and ``yaml`` configs have arbitrary depth.
 
-  .. code-block:: text
+   For example, when attempting to generate an ``sh`` config from a depth-2 ``yaml``:
+
+   .. code-block:: text
+
+      $ uw config realize --input-file config.yaml --output-format sh
+      [2024-01-23T23:02:42]    ERROR Cannot realize depth-2 config to type-'sh' config
+      Cannot realize depth-2 config to type-'sh' config
 
-     $ uw config realize --input-file config.yaml --output-format sh
-     [2024-01-10T21:46:00]    ERROR Cannot realize depth-2 config to type-'sh' config
-     Cannot realize depth-2 config to type-'sh' config
+.. note:: In recognition of the different sets of value types representable in each config format, ``uw`` supports two format-combination schemes:
```

Comment:
Just a preference, I think after looking at the documentation having back-to-back notes might seems like a lot. The second note to me seems like it can be un-noted and just standard text. Feel free to do what you think is best!